exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -185,11 +181,29 @@ impl JavaRuntimeProxy {
 }
 
 impl Runtime for JavaRuntimeProxy {
+    fn initialize(&mut self, blockchain: &Blockchain) {
+        // Store validator ID of the current node to use it in `afterCommit` later.
+        let node_public_key = &blockchain.service_keypair().0;
+        self.validator_id = Self::validator_id(&blockchain.snapshot(), node_public_key);
+
+        unwrap_jni(self.exec.with_attached(|env| {
+            let node_handle = to_handle(NodeContext::new(self.exec.clone(), blockchain.clone()));
```

Comment:
Good catch. Looks like it is indeed unnecessary for current implementation. I also renamed NodeContext to NodeProxy just for consistency with Java side.

Reply:
Ok, but in Java that's a proxy of a native object; here that's an actual object.

Reply:
Makes sense, renamed to `Node` :D
Naming is hard here, because the naming of `Blockchain` structure isn't stable yet in Core, so it might be changed in the future.